lib/index.js

```diff
@@ -6,111 +6,125 @@ const { getDungeonExpWeight } = require("./dungeonExpWeight");
 const { getSlayerWeight } = require("./slayerWeight");
 
 /**
- * Gets the player's raw weight. This makes no API requests.
- * Order of skills: enchanting, taming, alchemy, mining, farming, foraging, combat, fishing.
- * Order of slayers: zombie, spider, wolf, enderman.
- * @param {number[]} skillLevels - Array of skill levels in the order listed above. They all scale up to 60.
- * @param {number[]} skillXP - Array of skill XP in the order listed above.
- * @param {{"0": number, "1": number, "2": number, "3": number, "4": number, "5": number, "6": number, "7": number}} cataCompl - Object of catacombs completion, e.g. { "0": 13, "1": 37, "2": 32, ... }.
- * @param {{"1": number, "2": number, "3": number, "4": number, "5": number, "6": number}} mCataCompl - Object of master catacombs completion, same format as cataCompl.
- * @param {number} cataXP - Catacombs experience.
- * @param {number[]} slayerXP - Array of slayer experience amounts in the order listed above.
- * @returns {{total: number, slayer: number, skill: {overflow: number, base: number}, catacombs: {completion: {base: number, master: number}, experience: number}}}
+ * an object containing weighs
+ * @typedef {{total: number, slayer: number, skill: {overflow: number, base: number}, catacombs: {completion: {base: number, master: number}, experience: number}}} Weights
  */
-const getWeightRaw = (skillLevels, skillXP, cataCompl, mCataCompl, cataXP, slayerXP) => {
-    const [skillWeight, overflowWeight] = getSkillWeight(skillLevels, skillXP);
-    const [cataComplWeight, masterCataComplWeight] = getDungeonCompletionWeight(cataCompl, mCataCompl);
-    const cataExpWeight = getDungeonExpWeight(cataXP);
-    const slayerWeight = getSlayerWeight(slayerXP);
 
-    return {
-        total: (skillWeight + overflowWeight + cataComplWeight + masterCataComplWeight + cataExpWeight + slayerWeight),
-        skill: {
-            base: skillWeight,
-            overflow: overflowWeight
-        },
-        catacombs: {
-            completion: {
-                base: cataComplWeight,
-                master: masterCataComplWeight
-            },
-            experience: cataExpWeight
-        },
-        slayer: slayerWeight
-    };
-};
+/**
+ * an object containing weighs, the uuid and maybe username
```

Comment:
This comment should be uppercase, have a dot at the end and the typos should be fixed, weighs -> weights, uuid -> UUID.

Reply:
resolved by 39114f5 and 522d79e